Buildings/Resources/Scripts/OpenModelica/compareVars/Buildings.Experimental.RadiantControl.SlabTempSignal.OutputOnly.Validation.ForHiChi.mos

```diff
@@ -0,0 +1,4 @@
+compareVars :=
```

Comment:
This one looks empty, please populate for those models you decide are finished and will stay in the package you deliver.